gm4_liquid_tanks/data/gm4_standard_liquids/function/util/honey_casting/honey_rise.mcfunction

```diff
@@ -1,9 +1,9 @@
 #@s = gm4_honey_display
 #run from standard_liquids:util/honey_casting/scheduled_honey_rise
 
-teleport @s ~ ~0.0375 ~
 scoreboard players add @s gm4_lt_util 1
+execute if score @s gm4_lt_util matches ..12 teleport @s ~ ~0.060 ~
```

Comment:
Missing `run`

Reply:
Well thats a fun way to find out bolt will add implicit `runs` to execute chains